Cython/Compiler/FusedNode.py

```diff
@@ -923,6 +923,10 @@ def generate_execution_code(self, code):
                                     (self.resulting_fused_function.result(),
                                      self.__signatures__.result()))
             self.__signatures__.generate_giveref(code)
+            # TODO would "incref, giveref, gevenrate_disposal code" be more idomatic (but
```

Comment:
Typo in comment (`generate_disposal code`), but what I think you want here is `generate_post_assignment_code()`.